_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -226,6 +226,8 @@ function get_initial_state() {
 
 		$current_user_data = jetpack_current_user_data();
 
+		$status = new Status();
```

Comment:
Now that you're creating an instance here, shouldn't the line 239 be something like:

```php
'isActive' => $status->is_development_mode(),
```

Reply:
Good catch. Removing the instance in a948dac66 since Status is only being used once.